packages/react/src/components/SideMenu/SideMenuPositioner.tsx

```diff
@@ -77,40 +83,26 @@ export const SideMenuPositioner = <
         freezeMenu={props.editor.sideMenu.freezeMenu}
         unfreezeMenu={props.editor.sideMenu.unfreezeMenu}
       />
-    );
-  }, [block, props.editor, props.sideMenu]);
-
-  return (
-    <Tippy
-      appendTo={props.editor.domElement.parentElement!}
-      content={sideMenuElement}
-      getReferenceClientRect={getReferenceClientRect}
-      interactive={true}
-      visible={show}
-      animation={"fade"}
-      offset={offset}
-      placement={"left"}
-      popperOptions={popperOptions}
-      zIndex={1000}
-    />
+    </div>
   );
 };
 
-const offset: [number, number] = [0, 0];
-const popperOptions = {
-  modifiers: [
-    {
-      name: "flip",
-      options: {
-        fallbackPlacements: [],
-      },
-    },
-    {
-      name: "preventOverflow",
-      options: {
-        mainAxis: false,
-        altAxis: false,
-      },
-    },
-  ],
-};
+// TODO: anything from below needed?
```

Comment:
This is just to ensure the side menu doesn't change position if it overflows. In Floating UI I think the way to do this is to just omit `flip()` from the middleware.